ta/arch/arm/link_shlib.mk

```diff
@@ -19,6 +19,7 @@ cleanfiles += $(link-out-dir)/$(shlibuuid).ta
 
 shlink-ldflags  = $(LDFLAGS)
 shlink-ldflags += -shared -z max-page-size=4096
+shlink-ldflags += $(call ld-option,-z separate-loadable-segments)
```

Comment:
`cc-option` seems to be called with a second empty parameter. Why is this different?

Reply:
Good catch :) the empty parameter is simply useless and we {c,sh}ould have called `cc-option` with only one parameter... I realized it too late in fact.